combat-trainer.lic

```diff
@@ -2575,9 +2576,14 @@ class AttackProcess
 
     @use_overrides_for_aiming_trainables = settings.use_overrides_for_aiming_trainables
     echo(" @use_overrides_for_aiming_trainables: #{@use_overrides_for_aiming_trainables}") if $debug_mode_ct
-    
+
+    @firing_delay = settings.firing_delay || 15 # Delay for shooting aimed weapons.
```

Comment:
❓ Is this the same case as `loot_delay` above? I.e. it should instead be in `base.yaml`.